quetz/pkgstores.py

```diff
@@ -571,6 +604,7 @@ def __init__(self, config):
             token=self.token if self.token else None,
             cache_timeout=self.cache_timeout,
             default_location=self.region,
+            asynchronous=True,
```

Comment:
Do we also have to pass the `asynchronous=True` falg to the other `fsspec`-based package stores?

Reply:
That's a good question and observation. My hunch is that it should, but I'll check and most likely add that.